public/pages/MonitorDetails/components/MonitorOverview/utils/getOverviewStats.js

```diff
@@ -56,7 +58,7 @@ function getMonitorType(searchType) {
 function getMonitorLevelType(monitorType) {
   console.log(monitorType);
```

Comment:
Just saw this. Do we want to remove this `console.log` line?

Reply:
Good catch. I was actually just about to ask about this too. I think it'd be worthwhile to move the log to the `default` case, and reword it to something like "Unexpected monitor type".

Reply:
Per offline conversation, we'll resolve this in a separate PR by moving the `log` line to the `default` case, and will investigate possibly adding a toast popup to that case as well in the future.